pontoon/gpt/openai_connection.py

```diff
@@ -0,0 +1,56 @@
+from openai import OpenAI
+
+
+class OpenAIService:
+    def __init__(self):
+        self.client = OpenAI()
+
+    def get_translation(self, english_text, slovenian_text, formality):
+        # Define system messages for different translation tasks
+        system_messages = {
+            "informal": """You will be provided with a passage in English, along with its machine-generated translation in Slovenian. Your objective is to revise the Slovenian translation to ensure it utilizes simpler language. Please adhere to the following guidelines to achieve this:
+                    - Simplify Vocabulary: Use common, everyday words that are easily understood by a broad audience, avoiding technical jargon, idiomatic expressions, and complex terms.
```

Comment:
This feels too much? Why should it avoid idiomatic expressions?

Reply:
I was trying to make the text more accessible to a broader audience who may not understand specific idiomatic expressions because of the cultural nuances associated with them. But I see your point, the translation may be oversimplified.